makefiles/Makefile.docs-landing

```diff
@@ -60,13 +60,6 @@ next-gen-html:
 	else \
 		exit 0; \
 	fi \
```

Comment:
seems like this fi should also remove the extra `\` ?

Reply:
Thanks for catching! The behavior can be so weird.

This is not fine:
```
node --unhandled-rejections=strict test.js
if [ $$? -eq 1 ]; then \
  exit 1; \
else \
  exit 0; \
fi \
@echo "Hello world"
```

This is fine:
```
node --unhandled-rejections=strict test.js
if [ $$? -eq 1 ]; then \
  exit 1; \
else \
  exit 0; \
fi \
# idk
@echo "Hello world"
```

This is fine:
```
node --unhandled-rejections=strict test.js
if [ $$? -eq 1 ]; then \
  exit 1; \
else \
  exit 0; \
fi \
```